apps/communications/contacts/test/marionette/activities_test.js

```diff
@@ -94,7 +94,8 @@ marionette('Contacts > Activities', function() {
       smsSubject.launch(); // We open some app to start a Marionette session.
     });
 
-    test('open text/vcard activity (!allowSave) opens vCard in details view',
+    // Skipped due to bug 1138371
+    test.skip('open text/vcard activity (!allowSave) opens in details view',
```

Comment:
we cannot land code with test.skip, we have two options: modify them or remove them. I would prefer the former